plugins/chat/assets/javascripts/discourse/components/chat-channel-row.js

```diff
@@ -47,6 +49,9 @@ export default Component.extend({
   @discourseComputed("active", "channel.{id,muted}", "channel.focused")
   rowClassNames(active, channel, focused) {
     const classes = ["chat-channel-row", `chat-channel-${channel.id}`];
+    const channelUnreadCount =
+      this.currentUser.chat_channel_tracking_state[channel.id].unread_count;
+
```

Comment:
@jjaffeux, not sure how to fix the failing test, but it seems to have to do with this coming out as undefined in the `chat-channel-row` component test. I've tried setting it `beforeEach()` in the test but it doesn't seem to be working. Any ideas on how to resolve this?